moped-editor/src/views/projects/projectView/ProjectComments.js

```diff
@@ -96,20 +92,21 @@ const ProjectComments = props => {
   const [commentAddSuccess, setCommentAddSuccess] = useState(false);
   const [editingComment, setEditingComment] = useState(false);
   const [commentId, setCommentId] = useState(null);
+  const [displayNotes, setDisplayNotes] = useState({});
```

Comment:
if you you're gonna keep this state hook, i think it should initialize as an empty array or `null` instead of an empty object?

Reply:
good catch. Array seems best right? Since that's what we expect `displayNotes` to be?